draft-ietf-taps-impl.md

```diff
@@ -820,8 +820,10 @@ Cached protocol state is primarily used during Connection establishment for a si
 implementation's preference between several candidate Protocol Stacks. For example, if two IP address Endpoints are otherwise
 equally preferred, an implementation may choose to attempt a connection to an address for which it has a TCP Fast Open cookie.
 
-Applications must have a way to flush protocol cache state if desired. This may be necessary, for example, if
-application-layer identifiers rotate and clients wish to avoid linkability via trackable TLS tickets or TFO cookies.
+Applications must have a way to not use protocol cache state for certain
+connections, if desired. This may be necessary, for example, if application-
```

Comment:
I like the intent, however
(1) This wasn't clear: /if desired/  ... if who desires? The application, the TAPS system, the implementor. 
(2) I'm not a huge fan of /if desired/... /must/ even when it is lower case.
Could this be: Applications can request that the TAPS system ought not to use the cached protocol state for certain connections?

Reply:
One question on the phrase "not use cached protocol state". I read that as not using the cached state that is there when initiating a connection, but the taps system should also not cache state about this connection for use in later connections.  Not sure if that is captured by the wording or should be made explicit? It is clear in the update to the API section.
Otherwise looks good to me as well!